modules/dynamic-operator/Makefile

```diff
@@ -36,6 +36,8 @@ ifeq ($(USE_BUILD),true)
 endif
 COMMIT_SHA = $(shell git rev-parse HEAD)
 IMG_SHA_TAG ?= $(VERSION)-$(COMMIT_SHA)
+LDFLAGS = -ldflags="-X main.ver=$(VERSION)"
```

Comment:
It can configuring only for the specific step of Makefile, no global

Reply:
fixed